src/Toolchain/ToolchainEnv.ts

```diff
@@ -71,8 +71,10 @@ class Env implements BuilderJob {
     const rootJobs = this.workFlow.jobs.filter((j) => j.root === true);
     if (rootJobs.length > 0) {
       Logger.info(this.logTag, "Showing password prompt");
+      const helpMsg: string =
+        "It requires sudo authenticataion. Please enter your password.";
```

Comment:
```suggestion
        "It requires sudo authentication. Please enter your password.";
```

Reply:
How about using `Enter password for blah blah` help message?
The vscode basically shows a help message like this.
![a](https://github.com/Samsung/ONE-vscode/assets/7223627/c85f0fc1-3ece-4b42-8189-c8257a583f92)

Thank you for your contribution! :)

Reply:
Then let's use "Enter password for sudo authentication". (`Enter password for` from vscode, `sudo authentication` from ubuntu software updater`)